app/main.py

```diff
@@ -1,8 +1,17 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people.update({name: self})
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(i['name'], i['age']) for i in people]
+    for j in people:
```

Comment:
Could you use more descriptive names for variables?
You can use `person` instead of `i` and `j`

Reply:
Yes, you're right